agent.go

```diff
@@ -531,7 +536,12 @@ func (p *pod) listenToUdevEvents(done chan struct{}) {
 
 			// Ignore udev events for block devices, these are handled by another go-routine in waitForBlockDevice
 			// that mounts block devices at the expected location provided on the command line.
-			if d.Subsystem() == "block" {
+			if d.Subsystem() == blockSubsystem {
+				continue
+			}
+
+			if d.Subsystem() == cpuSubsystem {
+				p.onlineVCPU(d.Syspath())
```

Comment:
if this operation takes some time, you can consider doing it in a separate go-routine.

Reply:
this function only writes ```1``` in ```/sys/devices/system/cpu/cpuX/online```